mysql/tests/test_statements.py

```diff
@@ -525,6 +525,47 @@ def run_query(q):
     assert metric['dd_commands'] == expected_metadata_payload['commands']
 
 
+@pytest.mark.parametrize("disable_sql_obfuscation", [True, False])
```

Comment:
I think it would be better to pull out the 'logic' of the test and make the test inputs fully parameterized. I.e. 

Inputs:
- test query
- query signature
- disable obfuscation

Outputs:
- expected parameterized query